tests/test_custom_models.py

```diff
@@ -2032,6 +2137,49 @@ def test_requires_grad_oft_same_targets(self):
             "base_model.model.lin0.oft_r.adapter1",
         )
 
+    def test_requires_grad_boft_same_targets(self):
```

Comment:
Not a must, but nice to have: Same test but for different target layers.

Reply:
Added test test_requires_grad_boft_different_targets. Done.